sdk/containerregistry/container-registry/samples/v1-beta/typescript/src/uploadImage.ts

```diff
@@ -23,22 +22,22 @@ async function main() {
   );
 
   const config = Buffer.from("Sample config");
-  const uploadConfigResult = await client.uploadBlob(Readable.from(config));
+  const uploadConfigResult = await client.uploadBlob(config);
 
   const layer = Buffer.from("Sample layer");
-  const uploadLayerResult = await client.uploadBlob(Readable.from(layer));
+  const uploadLayerResult = await client.uploadBlob(layer);
 
   const manifest: OciImageManifest = {
     schemaVersion: 2,
     config: {
       digest: uploadConfigResult.digest,
-      size: config.byteLength,
+      sizeInBytes: config.byteLength,
       mediaType: "application/vnd.oci.image.config.v1+json",
     },
     layers: [
       {
         digest: uploadLayerResult.digest,
-        size: layer.byteLength,
+        sizeInBytes: layer.byteLength,
```

Comment:
what about  `layer.byteLength` ? it should come from uploadLayerResult

Reply:
Huh. These are auto-generated, the authoritative samples are in the `samples-dev` folder. Let me check why this hasn't been updated and regenerate. In the mean time use the ones in `samples-dev`.